release_dashboard/lib/widgets/apply_engine_cherrypicks.dart

```diff
@@ -0,0 +1,99 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'common/checkbox_substep.dart';
+
+enum SubstepEnum {
```

Comment:
avoid having the type in the name of the type. It's like saying `String fooString = 'foo';`